src/components/Main/Main.js

```diff
@@ -30,15 +31,48 @@ export class MainComponent {
 
   addEventListeners() {
     this.sidebarList.addEventListener("click", this.handleListClick.bind(this));
```

Comment:
bind handleListClick function in the constructor once. you should not call bind function every time event triggers

Reply:
actually not - it binds one time when addEventListeners function is being called